app/models/post.rb

```diff
@@ -310,7 +312,7 @@ def set_timestamps
     return if skip_edited
     self.edited_at = self.updated_at
     return if skip_tagged
-    return if replies.exists? && !status_changed?
+    return if replies.exists? && status != Post::STATUS_COMPLETE
```

Comment:
I think you might still need to check if the status changed here, as well as what it is?

Reply:
...oh. yes. I did... not entirely think that through, yeah xD